src/main/groovy/life/qbic/datamodel/datasets/datastructure/files/nanopore/DriftCorrectionLog.groovy

```diff
@@ -0,0 +1,32 @@
+package life.qbic.datamodel.datasets.datastructure.files.nanopore
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+
+/**
+ * A specialisation of a DataFile, represents an Oxford Nanopore Drift Correction file
+ *
+ * @author: Sven Fillinger
+ */
+class DriftCorrectionLog extends DataFile {
+
+    final private static String FILE_TYPE = "csv"
+
+    final private static String NAME_SCHEMA = $/drift_correction_.*/$
+
+    protected DriftCorrectionLog() {}
+
+    protected DriftCorrectionLog(String name, String relativePath) {
+        super(name, relativePath, FILE_TYPE)
+        validateName()
+    }
+
+    static DriftCorrectionLog create(String name, String relativePath) {
```

Comment:
Add `fileType` to `create` method.

Reply:
nope, the fileType is an intrinsic property of the specialisations of DataFile